azure-pipelines.yml

```diff
@@ -26,23 +26,17 @@ jobs:
 
     steps:
       - task: Gradle@2
+        name: 'clover'
         inputs:
-          workingDirectory: ''
-          gradleWrapperFile: 'gradlew'
-          gradleOptions: '-Xmx3072m'
-          javaHomeOption: 'JDKVersion'
-          jdkVersionOption: '1.8'
-          jdkArchitectureOption: 'x64'
-          publishJUnitResults: true
-          testResultsFiles: '**/TEST-*.xml'
-          tasks: 'build'
-
-      - task: PublishCodeCoverageResults@1
```

Comment:
I think you will still need `PublishCodeCoverageResults@1` task after `ReportGenerator`. Would be enough to just set `reportDirectory: '$(System.DefaultWorkingDirectory)/iofog-agent-daemon/build/reports/clover'` property.